docs/testing/runtime-logging.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: Debug your add-in with runtime logging
```

Comment:
Consider using title-casing as per https://review.learn.microsoft.com/en-us/help/platform/seo-meta-title?branch=main

Reply:
This seems like it should be adjusted across the board. I'll make an internal item to address that.